nifi-nar-bundles/nifi-extension-utils/nifi-prometheus-utils/src/main/java/org/apache/nifi/prometheus/util/PrometheusMetricsUtil.java

```diff
@@ -249,10 +249,8 @@ public static CollectorRegistry createNifiMetrics(NiFiMetricsRegistry nifiMetric
                 nifiMetricsRegistry.setDataPoint(portStatus.getBytesSent(), "AMOUNT_BYTES_SENT", instanceId, portComponentType, portComponentName, portComponentId, parentId);
                 nifiMetricsRegistry.setDataPoint(portStatus.getInputBytes(), "AMOUNT_BYTES_READ", instanceId, portComponentType, portComponentName, portComponentId, parentId);
                 nifiMetricsRegistry.setDataPoint(portStatus.getOutputBytes(), "AMOUNT_BYTES_WRITTEN", instanceId, portComponentType, portComponentName, portComponentId, parentId);
-                nifiMetricsRegistry.incrementCounter(status.getBytesRead(), "TOTAL_BYTES_READ", instanceId, portComponentType, portComponentName, portComponentId, parentId);
-                nifiMetricsRegistry.incrementCounter(status.getBytesWritten(), "TOTAL_BYTES_WRITTEN", instanceId, portComponentType, portComponentName, portComponentId, parentId);
-                nifiMetricsRegistry.incrementCounter(status.getBytesReceived(), "TOTAL_BYTES_RECEIVED", instanceId, portComponentType, portComponentName, portComponentId, parentId);
-                nifiMetricsRegistry.incrementCounter(status.getBytesSent(), "TOTAL_BYTES_SENT", instanceId, portComponentType, portComponentName, portComponentId, parentId);
+                nifiMetricsRegistry.incrementCounter(portStatus.getBytesReceived(), "TOTAL_BYTES_RECEIVED", instanceId, portComponentType, portComponentName, portComponentId, parentId);
+                nifiMetricsRegistry.incrementCounter(portStatus.getBytesSent(), "TOTAL_BYTES_SENT", instanceId, portComponentType, portComponentName, portComponentId, parentId);
```

Comment:
@mattyb149 in this instance we'll need to keep the TOTAL_BYTES_READ and TOTAL_BYTES_WRITTEN (lines previously on 252 and 253 however the portStatus.getInputBytes() and portStatus.getOutputBytes() should be used respectively as in lines 250 and 251

Reply:
Good catch, I saw the behavior while doing some additional testing. Since the second commit isn't valid, I'll just correct the original and force-push